news/2021-07-26-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: post
+title: "Performance Points & Star Rating Updates"
+date: 2021-07-27 10:00:00 +0000
+---
+
+A brand new set of changes to the performance points and star rating algorithms have just gone live - read on to discover what's new, and where things are headed in the future!
+
+![](/wiki/shared/news/2021-07-27-performance-points-star-rating-updates/star-rating.jpg)
+
+Thanks to the efforts of some very bright minds, a new batch of changes are live for both the osu! and osu!mania game modes. Read on below to check out all the changes stated in full (courtesy of a write-up from [spaceman_atlas](https://osu.ppy.sh/users/3035836)), including a number of graphs and charts to help things along.
+
+Please note: the current batch of changes contain things that will take effect immediately, as well as ones which will have no visible change for the moment but will lay out the groundwork for improvements in the future.
+
+Additionally, the changes below are split into "star rating" and "performance points" sections, but do note that anything that affects star rating will also impact performance points, as both are based off an algorithmic *skill* value which is dynamically generated from the beatmap being played.
+
+You may also wish to consult the [performance points](/wiki/Performance_points) and [star rating](/wiki/Beatmapping/Star_rating) articles to broaden your understanding of these surprisingly complicated topics.
+
+As a brief reminder/summary, the following values are a core part of the performance points algorithm:
+
+- **Aim**, which considers how difficult it is to consistently hit notes back-to-back during play (affected by things like approach rate, jumps, Flashlight, Hidden, Hard Rock and more)
+- **Speed**, which considers how quickly new elements are presented to the player during the course of the map (affected by things like the number of hit objects over time, BPM, Double Time and Half Time)
+- **Accuracy**, which considers the player's ability to stay on time rhythmically and how difficult this is to achieve (affected by things like Hidden, Hard Rock, Flashlight and more)
+- **Strain**, which considers how often and how long a player has to endure intense or technically demanding patterns during play (affected by things like streams, fast jumps and more)
+
+## Release Schedule
+
+Deployment of these changes will as usual, take some time to propagate out. As this batch of changes affects all game modes, it is likely to take longer than usual. If you are interested in following the progress we recommend you bookmark this page and check back on this section, as it will be updated as forward progress is made.
```

Comment:
the deployment will propagate or the changes will propagate? (maybe doesn't matter, but this reads better)

```suggestion
Deployment of these changes will as usual take some time, but since this batch of changes affects all game modes, they will likely take longer than usual to propagate. If you are interested in following the progress we recommend you bookmark this page and check back on this section, as it will be updated when forward progress is made.
```

Reply:
Are you sure about "when" instead of "as"?

Reply:
deliberated on this a bit, because they mean the same thing in practice, but i guess it's fine to keep the emphasis on the gradual-ness